pages/about/index.tsx

```diff
@@ -32,6 +33,64 @@ interface TeamSection {
   members: string;
 }
 
+type ArticleType = {
+  title: string;
+  url: string;
+  sourceName: string;
+  lang: string | null;
+  publishDate: string;
+};
+
+type ArticlesListProps = {
+  articles: ArticleType[];
+};
+
+type ArticleProps = {
+  article: ArticleType;
+};
+
+const flagEmojis = {
+  uk: '🇺🇦',
+  cs: '🇨🇿',
+  de: '🇩🇪',
+  en: '🇬🇧',
+  es: '🇪🇸',
+  fr: '🇫🇷',
+  it: '🇮🇹',
+  pl: '🇵🇱',
+  ru: '🇷🇺',
+  sk: '🇸🇰',
+};
+
+type ObjectKey = keyof typeof flagEmojis;
+
+const Article = ({ article }: ArticleProps): JSX.Element => {
+  return (
+    <div className="w-full lg:max-w-full lg:flex px-2 py-1 my-1 rounded shadow border">
+      <a className="hover:text-primary-blue" href={article.url} target="_blank" rel="noreferrer">
+        {article.title}
+      </a>
+      <div className="font-light">
+        <span className="inline-block lg:pl-2 pr-2">{article.sourceName}</span>
+        <span className="inline-block pr-2">{new Date(article.publishDate).toLocaleDateString()}</span>
+        {article.lang && <span role="img">{flagEmojis[article.lang as ObjectKey]}</span>}
```

Comment:
Using `as` typecasting is best to avoid if you can :) In this case you can solve the typescript error by explicitly typing flagEmojis more broadly: 
`const flagEmojis: Record<string, string> = {...`

Then you can get rid of `ObjectKey` :)

Reply:
Thank you, I made `flagEmojis` explicitly typed.